tests/urlpatterns_reverse/urls_template_response_error_handlers.py

```diff
@@ -0,0 +1,11 @@
+# Used by the CustomErrorHandlerTests test case.
+
+from django.conf.urls import url
+
+from .views import template_response_error_handler, permission_denied_view
```

Comment:
The isort issue is related to this import. Can you please swap `template_response_error_handler` and `permission_denied_view`.